src/app/feedback-widget/feedback-widget.component.css

```diff
@@ -0,0 +1,25 @@
+*{
```

Comment:
Using * is not the best idea mostly for performance and predictability reasons, just remove this whole section